web/client/translations/data.en-US

```diff
@@ -1889,6 +1889,39 @@
                 "editMedia": "Change media source",
                 "remove": "Remove"
             },
+            "builder": {
+                "defaults": {
+                    "htmlTitle": "<h1 style='text-align:center'>Insertar título</h1><p style='text-align:center'><em>sottotitolo</em></p>",
+                    "htmlSample": "<p>Inserisci del testo qui...</p>",
+                    "titleTitle": "Title Section",
```

Comment:
We should check if we should add "section" on every title. I think we shouldn't

Reply:
mmmh, as far as i know at the moment we have only sections of Titles, not sure if in the future we will have somethign else.. anyway it's a default value which should be similar to the type?